jsettlers.main.swing/src/jsettlers/main/swing/menu/openpanel/OpenPanel.java

```diff
@@ -103,22 +111,23 @@ public OpenPanel(final List<MapLoader> maps, IMapSelectedListener mapSelectedLis
 	 * @param cellRenderer
 	 *            Cell renderer to use
 	 */
-	public OpenPanel(final List<MapLoader> maps, final IMapSelectedListener mapSelectedListener, final ListCellRenderer<MapLoader> cellRenderer) {
+	public OpenPanel(final List<? extends MapLoader> maps, final IMapSelectedListener mapSelectedListener,
+			final ListCellRenderer<MapLoader> cellRenderer) {
 		setMapLoadersWithoutSearchChanged(maps);
 		setLayout(new BorderLayout());
 
 		initFilter();
 
 		this.searchTextField = new SearchTextField();
-		searchTextField.putClientProperty(ELFStyle.KEY, ELFStyle.TEXT_DEFAULT);
+		this.searchTextField.putClientProperty(ELFStyle.KEY, ELFStyle.TEXT_DEFAULT);
```

Comment:
In general, I don't want to enforce writing this everywhere. I prefer writing it, where I need it and if I write it once in a method, I write it everywhere in that method... I'm open for discussion on this (maybe in an issue to create a wiki entry for this).